core/src/main/scala/cats/syntax/bifoldable.scala

```diff
@@ -1,4 +1,29 @@
 package cats
 package syntax
 
+import scala.language.implicitConversions
+
 trait BifoldableSyntax extends Bifoldable.ToBifoldableOps
+
+private[syntax] trait BiFoldableSyntaxBinCompat0 {
+  implicit final def catsSyntaxBiFoldOps[F[_, _], A, B](fa: F[A, B]): BifoldBifoldableOps[F, A, B] =
```

Comment:
It's all implicit or private, so it doesn't matter that much, but everything else is `bifold`, not `biFold`.  Would b egood to remain consistent.